src/style.css

```
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star + .stars__star {
+  margin: 0 0 0 4px;
+}
```

You can safely add right margin to right star here
```suggestion
.stars__star {
  margin-right: 4px;
}
```